suite-native/module-authorize-device/src/components/connect/ConnectDeviceScreenHeader.tsx

```diff
@@ -17,6 +17,10 @@ import { Translation } from '@suite-native/intl';
 import { selectIsDeviceDiscoveryActive } from '@suite-common/wallet-core';
 
 import { ConnectingTrezorHelp } from './ConnectingTrezorHelp';
```

Comment:
there are some lint and formatting errors

Reply:
1b6ac2759d